server/athenian/api/models/state/versions/79c33483be7d_use_jsonb.py

```diff
@@ -0,0 +1,120 @@
+"""Use JSONB
+
+Revision ID: 79c33483be7d
+Revises: 9172948308c9
+Create Date: 2022-07-28 09:17:55.121271+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "79c33483be7d"
+down_revision = "9172948308c9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("account_features") as bop:
+        bop.alter_column(
+            "parameters",
+            existing_type=sa.JSON(),
+            type_=postgresql.JSONB(astext_type=sa.Text()).with_variant(sa.JSON(), "sqlite"),
+            existing_nullable=True,
+        )
+    with op.batch_alter_table("features") as bop:
+        bop.alter_column(
+            "default_parameters",
+            existing_type=sa.JSON(),
+            type_=postgresql.JSONB(astext_type=sa.Text()).with_variant(sa.JSON(), "sqlite"),
+            existing_nullable=False,
+            existing_server_default=sa.text("'{}'::json"),
+        )
+    with op.batch_alter_table("repository_sets") as bop:
+        bop.alter_column(
+            "items",
+            existing_type=sa.JSON(),
+            type_=postgresql.JSONB(astext_type=sa.Text()).with_variant(sa.JSON(), "sqlite"),
+            existing_nullable=False,
+        )
+    with op.batch_alter_table("shares") as bop:
+        bop.alter_column(
+            "id",
+            existing_type=sa.INTEGER(),
+            type_=sa.BigInteger().with_variant(sa.Integer(), "sqlite"),
+            existing_nullable=False,
+            autoincrement=True,
+        )
+    with op.batch_alter_table("team_goals") as bop:
+        bop.alter_column(
+            "target",
+            existing_type=sa.JSON(),
+            type_=postgresql.JSONB(astext_type=sa.Text()).with_variant(sa.JSON(), "sqlite"),
+            existing_nullable=False,
+        )
+    with op.batch_alter_table("teams") as bop:
+        bop.alter_column(
+            "members",
+            existing_type=sa.JSON(),
+            type_=postgresql.JSONB(astext_type=sa.Text()).with_variant(sa.JSON(), "sqlite"),
+            existing_nullable=False,
+        )
```

Comment:
Question: did you test this? Will we have jsonb for real now?

Reply:
yes
```
$ psql -p 5432 -U api -h localhost -d state -c '\d team_goals' | fgrep target
 target     | json                     |           | not null | 
(api310) gaetano@deneb:~/prog/athenian-api/server$ alembic upgrade head
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade 9172948308c9 -> 58582455884b, Use JSONB
(api310) gaetano@deneb:~/prog/athenian-api/server$ psql -p 5432 -U api -h localhost -d state -c '\d team_goals' | fgrep target
 target     | jsonb                    |           | not null | 
(api310) gaetano@deneb:~/prog/athenian-api/server$ alembic downgrade 917
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running downgrade 58582455884b -> 9172948308c9, Use JSONB
(api310) gaetano@deneb:~/prog/athenian-api/server$ psql -p 5432 -U api -h localhost -d state -c '\d team_goals' | fgrep target
 target     | json                     |           | not null | 
 ```